artifactory/cli.go

```diff
@@ -621,15 +621,98 @@ func GetCommands() []cli.Command {
 	}
 }
 
-func getGlobalConfigFlag() []cli.Flag {
+func getBaseBuildToolsConfigFlags() []cli.Flag {
 	return []cli.Flag{
 		cli.BoolFlag{
 			Name:  "global",
-			Usage: "[Default: false] Set to true, if you'd like to configuration to be global (for all projects). Specific projects can override the global configuration.` `",
+			Usage: "[Default: false] Set to true if you'd like the configuration to be global (for all projects). Specific projects can override the global configuration.` `",
+		},
+		cli.BoolTFlag{
+			Name:  "interactive",
+			Usage: "[Default: true] Set to false if you do not want the config command to be interactive.` `",
```

Comment:
[Default: true] --> [Default: true, if CI is not set]
Also I suggest adding the following to the option description:
If the CI environment variable is set to true, the default value of this option is false. Otherwise, it is true.

Reply:
I'm removing this flag altogether